fe1-web/ingestion/Handler.ts

```diff
@@ -19,7 +19,7 @@ function handleLaoCreateMessages(msg: ExtendedMessage) : boolean {
   // processing the lao/create message:
   // - we either connect to the LAO (if there's no active connection) OR
   // - we simply add it to the list of known LAOs
-  handleLaoMessage(msg);
+  handleMessage(msg);
```

Comment:
... and just call `messageRegistry.handleMessage` from here.
We would need the `messageRegistry` to be a variable we can set from outside this module, though